src/components/Header/Header.tsx

```diff
@@ -5,22 +5,67 @@ import styles from './Header.module.scss'
 import logo from '../../assets/images/logo.png'
 import GithubIcon from '@/assets/icons/GithubIcon'
 import AddIcon from '@/assets/icons/AddIcon'
+import Image from 'next/image'
 import Link from 'next/link'
-import { useSession, signIn, signOut } from 'next-auth/react'
+import { useSession, signIn } from 'next-auth/react'
+import { usePathname } from 'next/navigation'
+import { useRouter } from 'next/navigation'
 
 const Header = () => {
   const { data: session, status } = useSession()
+  const name = session?.user?.name
+  const avatar = session?.user?.image
+  const pathname = usePathname()
+  const router = useRouter()
 
-  if (status === 'authenticated') {
+  if (
+    status === 'authenticated' &&
+    pathname === ('/my-profile' || '/create-profile')
```

Comment:
**nitpick:** This is not good solution, as it hardcodes pages urls that can change in time. My suggestion is to create different component, and `/my-profile`, and `/create-profile` can just have different layout component than homepage.